src/main/java/salesforce/ui/pages/genericTabs/TabObjectsLightningPage.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2019 Jalasoft.
+ *
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Jalasoft.
+ */
+
+package salesforce.ui.pages.genericTabs;
+
+import core.selenium.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.StaleElementReferenceException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import salesforce.ui.pages.AppPageFactory;
+import salesforce.ui.pages.lead.DeleteObjectLightningPopup;
+import salesforce.ui.pages.opportunity.AbstractOpportunityPage;
+
+/**
+ * Defines OpportunitiesPopUpLightning.
+ *
+ * @author Alan Escalera.
+ * @version 1.0 19 March 2020.
+ */
+public class TabObjectsLightningPage extends AbstractTabObjectsPage {
+    @FindBy(css = "div[title='New']")
+    private WebElement newButton;
+
+    @FindBy(css = "div[class='triggerLinkTextAndIconWrapper slds-p-right--x-large']")
+    private WebElement opportunityListButton;
+
+    protected static final String NAME_OBJECT = "a[title='%s']";
+    private static final String OBJECT_ORDERED_LIST_PARTIAL_LOCATOR = "//li[contains(.,'%s')]";
+    private WebElement nameOpportunityTable;
+
+    @Override
+    protected void waitUntilPageObjectIsLoaded() {
+        webDriverWait.until(ExpectedConditions.elementToBeClickable(newButton));
+    }
+
+    @Override
+    public void displayList(final String listName) {
+        String opportunityOrderedListLocator;
+        opportunityListButton.click();
+        opportunityOrderedListLocator = String.format(OBJECT_ORDERED_LIST_PARTIAL_LOCATOR, listName);
+        webDriverWait.until(ExpectedConditions.elementToBeClickable(webDriver.findElement(
+                By.xpath(opportunityOrderedListLocator))));
+        webDriver.findElement(By.xpath(opportunityOrderedListLocator)).click();
+    }
+
+    @Override
+    public AbstractOpportunityPage selectObjectByName(final String opportunityName) {
+        String opportunityNameXpath = String.format(NAME_OBJECT, opportunityName);
+        nameOpportunitySelected = WebDriverManager.getInstance().getWebDriver().findElement(By
+                .cssSelector(opportunityNameXpath));
+        nameOpportunitySelected.click();
+        return AppPageFactory.getOpportunityPage();
+    }
+
+    @Override
+    public void clickOnNewButton() {
+        newButton.click();
+    }
+
+    @Override
+    public void clickOnDeleteButton(final String nameObject) {
+        final String objectTabBaseLocator = "//a[@title='%s']//../../..//a[contains(@class,'rowActions')]";
+        final String deleteOptionLocator = "//div[contains(@class,'branding-actions')]//a[contains("
+                + "@title,'Delete')]";
```

Comment:
put this xpath after 37 line

Reply:
What is the reason if I don't use that in another method?

Reply:
We could use it in another features

Reply:
But it is not the case now, another can do that if it is necessary

Reply:
It is better to put now, if someone use en next features, maybe will create a new one the same